tests/lib/prepare.sh

```diff
@@ -737,14 +866,18 @@ fi
 EOF
     chmod +x "$IMAGE_HOME/reflash.sh"
 
+    DEVPREFIX=""
+    if is_core20_system; then
```

Comment:
I think we could actually always get away with adding /boot/vmlinuz, since those are valid symlinks on 18.04 as well (I think on 16.04 as well but I would need to check)

Reply:
Actually we can't, so this is the right thing to do